### PR TITLE
UI added

### PR DESCRIPTION
**Pull request by adCBueno**
Detail added with fragments for information and evolution, tabs incluted

### CODE REVIEW

File: PokeAppAndroid/View/PokemonDetailFragment.cs [hunk 1]
```diff
@@ -52,6 +100,14 @@ public void updatePokemonImage(Result<byte[]> image)
             pokemonImage.SetImageBitmap(bitmap);
```
Comment (YawarValeriano) [inline PokeAppAndroid/View/PokemonDetailFragment.cs:106]: Use picasso for images in android

File: PokeAppAndroid/View/PokemonInfoFragment.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+            controller = IocContainer.GetDependency<IPokemonDetailController>();
+            controller.listener = this;
+        }
+
+
+        public override Android.Views.View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
+        {
+            Android.Views.View view = inflater.Inflate(Resource.Layout.fragment_info_detail, container, false);
+            //secondaryPokemonSprite = view.FindViewById<ImageView>(Resource.Id.secondaryPokemonSprite);
```
Comment (YawarValeriano) [inline PokeAppAndroid/View/PokemonInfoFragment.cs]: Remove code comments

File: PokeAppAndroid/View/PokemonInfoFragment.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        {
+            throw new NotImplementedException();
+        }
+
+        public void updateSecondaryPokemonSprite(Result<byte[]> image)
+        {
+            if (image.IsFailure) return;
+            byte[] imageByte = image.Value;
+            Bitmap bitmap = BitmapFactory.DecodeByteArray(imageByte, 0, imageByte.Length);
+            secondaryPokemonSprite.SetImageBitmap(bitmap);
```
Comment (YawarValeriano) [inline PokeAppAndroid/View/PokemonInfoFragment.cs:73]: same about picasso

**Review by YawarValeriano — CHANGES_REQUESTED** [review]

**Review by YawarValeriano — CHANGES_REQUESTED** [review]
Also, there is a pokeball xml to use as resource

**Review by rodrigoschar — APPROVED** [review]

File: PokeAppAndroid/Resources/drawable/bluearrow.png
[the remark this answers is not in the document]
Comment (adCBueno) [reply PokeAppAndroid/Resources/drawable/bluearrow.png:1]: Sure. Those are the arrows in evolution fragment
Captura de pantalla 2023-04-10 a la(s) 13 35 54

File: PokeAppAndroid/Resources/values/styles.xml [hunk 4]
```diff
@@ -43,5 +43,10 @@
       <style name="DefaultImageMarginBottom">
         <item name="android:layout_marginBottom">10dp</item>
       </style>
+
+    <style name="pokemonDetailBackground">        
+        <item name="android:background">@drawable/pokeball</item>
+        <item name="android:backgroundColor">@color/bug</item>
```
Comment (jimibilibob) [inline PokeAppAndroid/Resources/values/styles.xml]: I guess the property backgroundColor es invalid.

**Review by jimibilibob — CHANGES_REQUESTED** [review]

File: PokeAppAndroid/Resources/values/strings.xml [hunk 5]
```diff
@@ -11,5 +11,11 @@
     <string name="passwordText">Password:</string>    
     <string name="usernamePlaceholder">Insert you email</string>
     <string name="passwordPlaceholder">Insert your password</string>
+    <string name="informationButton">Information</string>
+    <string name="evolutionButton">Evolution</string>
+    <string name="firstEvolution">Evolution 1</string>
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply PokeAppAndroid/Resources/values/strings.xml:18]: Sure

File: PokeAppiOS/PokeAppiOS.csproj [hunk 6]
```diff
@@ -157,6 +157,7 @@
     <ItemGroup>
         <None Include="Info.plist" />
         <None Include="Entitlements.plist" />
+        <None Include=".DS_Store" />
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply PokeAppiOS/PokeAppiOS.csproj]: Weird, I didn't added but googled and it's a normal file that Visual Studio uses and should ignore, this time it didn't but already deleted it

**Review by jimibilibob — APPROVED** [review]

File: PokeAppAndroid/View/PokemonInfoFragment.cs (hunk 3 above)
Comment (YawarValeriano) [reply PokeAppAndroid/View/PokemonInfoFragment.cs:73]: Solved since a new library will be implemented

**Review by YawarValeriano — APPROVED** [review]

**Pull request merged**